### PR TITLE
Updated BT samples to use the BT SDK 1.1.1

### PR DESCRIPTION
**Pull request by fkavache**
Updated BT samples to use the Body Tracking SDK 1.1.1
Fixed missing k4arecord library link in simple_3d_viewer cmake
Fixed samples project need for submodule initialization in order to build
Fixed no cmake for some samples

### CODE REVIEW

File: body-tracking-samples/sample_helper_libs/sample_shared_lib/DigitalSignalProcessing.cpp [hunk 1]
```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "DigitalSignalProcessing.h"
```
Comment (scotthsu98052) [inline body-tracking-samples/sample_helper_libs/sample_shared_lib/DigitalSignalProcessing.cpp:4]: I need some background info. Why is the code needed now? Why original sample code can be built without them?

Comment (scotthsu98052) [reply body-tracking-samples/sample_helper_libs/sample_shared_lib/DigitalSignalProcessing.cpp]: The update makes sense to me.

File: body-tracking-samples/extern/nlohmann_json/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+include(FetchContent)
+
+FetchContent_Declare(nlohmann_json
```
Comment (scotthsu98052) [inline body-tracking-samples/extern/nlohmann_json/CMakeLists.txt:6]: Still, please teach me how nlohmann json worked before without this change?

Comment (fkavache) [reply body-tracking-samples/extern/nlohmann_json/CMakeLists.txt:6]: We are using nuget package for VS
`<package id="nlohmann.json" version="3.7.0" targetFramework="native" />`

Similarly we are using nuget for GLFW for simple_3d_viewer.
`<package id="glfw" version="3.3.0" targetFramework="native" />`
and for cmake we have glfw as a git submodule.

**Review by scotthsu98052 — APPROVED** [review]

**Pull request merged**